### DISCUSSION TITLE
OKD change notes info

### DISCUSSION THREAD

**Discussion by pkqsun** [okd-project/okd#1695] [General]
Hi,

Just a free talk if I could got the OKD release change notes info from somewhere.
Such as the change history, example: 
4.13.0-0.okd-2023-08-04-164726 changes since 4.13.0-0.okd-2023-07-23-051208.
4.13.0-0.okd-2023-07-23-051208 changes since 4.13.0-0.okd-2023-07-09-062029

**Comment by vrutkovs** [comment]
release-controller has this information:
https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.13.0-0.okd-2023-08-04-164726?from=4.13.0-0.okd-2023-07-23-051208
https://amd64.origin.releases.ci.openshift.org/releasetag/4.13.0-0.okd-2023-07-23-051208?from=4.13.0-0.okd-2023-07-09-062029

**Reply by pkqsun** [reply]
Thanks @vrutkovs.
That's what I want to check.
